picard/ui/mainwindow.py

```diff
@@ -1332,32 +1329,44 @@ def update_actions(self):
         can_autotag = False
         can_submit = False
         can_extract = False
-        single = self.selected_objects[0] if len(self.selected_objects) == 1 else None
+
+        try:
+            single = self.selected_objects[0]
+        except IndexError:
+            single = None
```

Comment:
This is different. `single` was only set if exactly one object had been selected. It is being used to enable actions that should only be enabled for a single selection.

Reply:
Obviously, I missed that, I'll rework that.

Reply:
This change was removed; code rebased.